### ISSUE TITLE
[Bug] TOTK Crashes When Starting

### ISSUE THREAD

**Issue by snl25** [Ryujinx/Ryujinx#5458]
### Description of the issue

Tears of The Kingdom crashes the entire emulator when trying to start it. Every other game I have tried boots up fine. I have tried looking at the error codes but couldn't find the exact issue that is causing it.

### Reproduction steps

Booting up TOTK causes the entire emulator to crash.

### Log file

00:00:00.050 |I| Configuration LogValueChange: ResScale set to: 1
00:00:00.056 |I| Configuration LogValueChange: ResScaleCustom set to: 1.5
00:00:00.056 |I| Configuration LogValueChange: MaxAnisotropy set to: -1
00:00:00.057 |I| Configuration LogValueChange: AspectRatio set to: Fixed16x9
00:00:00.058 |I| Configuration LogValueChange: BackendThreading set to: Auto
00:00:00.058 |I| Configuration LogValueChange: GraphicsBackend set to: Vulkan
00:00:00.059 |I| Configuration LogValueChange: PreferredGpu set to: 0x10DE_0x2187
00:00:00.060 |I| Configuration LogValueChange: AntiAliasing set to: None
00:00:00.061 |I| Configuration LogValueChange: ScalingFilter set to: Bilinear
00:00:00.061 |I| Configuration LogValueChange: ScalingFilterLevel set to: 50
00:00:00.065 |I| Configuration LogValueChange: EnableDockedMode set to: False
00:00:00.076 |I| Configuration LogValueChange: EnableVsync set to: True
00:00:00.077 |I| Configuration LogValueChange: EnableShaderCache set to: True
00:00:00.077 |I| Configuration LogValueChange: EnableTextureRecompression set to: False
00:00:00.077 |I| Configuration LogValueChange: EnableMacroHLE set to: True
00:00:00.077 |I| Configuration LogValueChange: EnablePtc set to: True
00:00:00.077 |I| Configuration LogValueChange: EnableInternetAccess set to: False
00:00:00.077 |I| Configuration LogValueChange: EnableFsIntegrityChecks set to: True
00:00:00.077 |I| Configuration LogValueChange: FsGlobalAccessLogMode set to: 0
00:00:00.078 |I| Configuration LogValueChange: AudioBackend set to: OpenAl
00:00:00.078 |I| Configuration LogValueChange: AudioVolume set to: 1
00:00:00.079 |I| Configuration LogValueChange: MemoryManagerMode set to: HostMappedUnsafe
00:00:00.079 |I| Configuration LogValueChange: ExpandRam set to: False
00:00:00.079 |I| Configuration LogValueChange: IgnoreMissingServices set to: False
00:00:00.079 |I| Configuration LogValueChange: UseHypervisor set to: True
00:00:00.084 |N| Application PrintSystemInfo: Ryujinx Version: 1.1.962
00:00:00.089 |N| Application Print: Operating System: Microsoft Windows 10.0.22000 (X64)
00:00:00.089 |N| Application Print: CPU: Intel(R) Core(TM) i5-10400F CPU @ 2.90GHz ; 12 logical
00:00:00.090 |N| Application Print: RAM: Total 16154 MiB ; Available 7935 MiB
00:00:00.094 |N| Application PrintSystemInfo: Logs Enabled: Info, Warning, Error, Guest, Stub
00:00:00.094 |N| Application PrintSystemInfo: Launch Mode: UserProfile
00:00:00.964 |I| Application FixExtraDataInSpaceId: Fixed extra data for save data 0x2 in the User save data space
00:00:01.606 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:00.107 |I| Gpu : Backend Threading (Auto): True
00:00:00.352 |N| Application LoadApplication: Using Firmware Version: 16.0.3
00:00:00.352 |I| Application LoadApplication: Loading as XCI.
00:00:00.391 |I| ModLoader QueryContentsDir: Searching mods for Title 0100F2C0115B6000
00:00:00.393 |I| ModLoader QueryContentsDir: Searching mods for Title 0100F2C0115B6000
00:00:00.398 |I| Loader Load: Loading rtld...
00:00:00.408 |I| Loader PrintRoSectionInfo: rtld:
    Module: nnrtld
00:00:00.408 |I| Loader Load: Loading main...
00:00:01.146 |I| Loader PrintRoSectionInfo: main:
    Module: D:\home\Project\EX-King\App\Rom\NX64\Product_Optimize\code\EX-King.nss
    SDK Libraries: SDK MW+Nintendo+NintendoWare_Vfx2-15_3_1-Release
                   SDK MW+Nintendo+NintendoWare_Font-15_3_1-Release
                   SDK MW+Nintendo+NintendoWare_Ui2d-15_3_1-Release
                   SDK MW+Nintendo+NintendoWare_G3d2-15_3_1-Release
                   SDK MW+Nintendo+NintendoWare_Meshoptimizer_For_MeshCodec-0_15_0-Release
                   SDK MW+Microsoft+Havok_Cloth-2022_1
                   SDK MW+Microsoft+Havok_Animation-2022_1
                   SDK MW+Microsoft+Havok_AI-2022_1
00:00:01.147 |I| Loader Load: Loading subsdk0...
00:00:01.207 |I| Loader PrintRoSectionInfo: subsdk0:
    Module: multimedia
    SDK Libraries: SDK MW+Nintendo+NintendoSDK_movie-15_3_1-Release
00:00:01.207 |I| Loader Load: Loading sdk...
00:00:01.360 |I| Loader PrintRoSectionInfo: sdk:
    Module: nnSdk
    FS SDK Version: 15.3.1
    SDK Libraries: SDK MW+Nintendo+NintendoSDK_libz-15_3_1-Release
                   SDK MW+Nintendo+NintendoSdk_nnSdk-15_3_1-Release
                   SDK MW+Nintendo+NintendoSdk_BuildTarget-64Bit
                   SDK MW+Nintendo+NintendoSDK_NVN-15_3_1-Release
00:00:01.492 |I| Ptc Initialize: Initializing Profiled Persistent Translation Cache (enabled: True).
00:00:01.505 |I| Loader LoadNsos: Loading image 0 at 0x0000000008000000...
00:00:01.507 |I| Loader LoadNsos: Loading image 1 at 0x0000000008004000...
00:00:01.536 |I| Loader LoadNsos: Loading image 2 at 0x000000000c720000...
00:00:01.539 |I| Loader LoadNsos: Loading image 3 at 0x000000000cdc9000...
00:00:01.548 |I| ModLoader LoadCheats: Build ids found for title 0100F2C0115B6000:
    7501ABFE55FA41CFFEB46BD619BDCBF45B9CAE3A000000000000000000000000
    6F32C68DD3BC7D77AA714B80E92A096A737CDA77000000000000000000000000
    A6BB7C6FFA9673769FA74ED3D7B054191F25D29C000000000000000000000000
    B9046C31EB5D31271BE970FE732D38DF49C6AA21000000000000000000000000
00:00:01.551 |I| Application EnsureSaveData: Ensuring required savedata exists.
00:00:01.630 |I| Loader Start: Application Loaded: The Legend of Zelda: Tears of the Kingdom v1.2.0 [0100f2c0115b6000] [64-bit]
00:00:01.647 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair, JoyconLeft, JoyconRight' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:02.077 |N| GUI.RenderLoop Gpu PrintGpuInformation: NVIDIA NVIDIA GeForce GTX 1650 SUPER (Vulkan v1.3.242, Driver v536.40.0.0)
00:00:02.092 |I| GPU.MainThread Gpu LoadShaders: Loading 1 shaders from the cache...
00:00:02.214 |I| GPU.MainThread Gpu LoadShaders: Shader cache loaded.
00:00:02.871 |S| HLE.OsThread.22 ServiceAm GetAppletResourceUserId: Stubbed. {appletResourceUserId: 1}
00:00:02.998 |S| HLE.OsThread.22 ServiceAm AcquireForegroundRights: Stubbed. 
00:00:03.005 |S| HLE.OsThread.22 ServiceAm SetFocusHandlingMode: Stubbed. {unknownFlag1: False ; unknownFlag2: False ; unknownFlag3: True}
00:00:03.008 |S| HLE.OsThread.22 ServiceAm SetOutOfFocusSuspendingEnabled: Stubbed. {outOfFocusSuspendingEnabled: False}
00:00:03.095 |S| HLE.OsThread.22 ServicePctl Initialize: Stubbed. 
00:00:03.648 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair, JoyconLeft, JoyconRight' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:05.649 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair, JoyconLeft, JoyconRight' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:07.099 |S| HLE.OsThread.22 ServiceAm SetFocusHandlingMode: Stubbed. {unknownFlag1: True ; unknownFlag2: False ; unknownFlag3: True}
00:00:07.100 |S| HLE.OsThread.22 ServiceAm SetOutOfFocusSuspendingEnabled: Stubbed. {outOfFocusSuspendingEnabled: False}
00:00:07.101 |S| HLE.OsThread.22 ServiceAm SetRestartMessageEnabled: Stubbed. {restartMessageEnabled: True}
00:00:07.102 |S| HLE.OsThread.22 ServiceAm SetOperationModeChangedNotification: Stubbed. {operationModeChangedNotification: True}
00:00:07.103 |S| HLE.OsThread.22 ServiceAm SetPerformanceModeChangedNotification: Stubbed. {performanceModeChangedNotification: True}
00:00:07.526 |S| HLE.OsThread.10 ServiceNv FinishInitialize: Stubbed. 
00:00:07.557 |I| HLE.OsThread.22 SurfaceFlinger CreateLayerFromId: Creating layer 1
00:00:07.558 |I| HLE.OsThread.22 SurfaceFlinger CreateLayerFromId: Creating layer 2
00:00:07.651 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair, JoyconLeft, JoyconRight' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:07.695 |S| HLE.OsThread.10 ServiceNv InitializeEx: Stubbed. 
00:00:07.748 |S| HLE.OsThread.10 ServiceNv GetActiveSlotMask: Stubbed. 
00:00:07.818 |S| HLE.OsThread.10 ServiceNv SetNvMapFd: Stubbed. 
00:00:07.820 |S| HLE.OsThread.10 ServiceNv AllocGpfifoEx2: Stubbed. 
00:00:07.821 |S| HLE.OsThread.10 ServiceNv AllocObjCtx: Stubbed. 
00:00:07.823 |S| HLE.OsThread.10 ServiceNv SetErrorNotifier: Stubbed. 
00:00:07.823 |S| HLE.OsThread.10 ServiceNv SetUserData: Stubbed. 
00:00:07.824 |S| HLE.OsThread.10 ServiceNv SetTimeout: Stubbed. 
00:00:07.825 |S| HLE.OsThread.10 ServiceNv SetTimeslice: Stubbed. 
00:00:07.831 |S| HLE.OsThread.10 ServiceNv ZcullBind: Stubbed. 
00:00:07.975 |S| HLE.OsThread.22 ServiceAcc InitializeApplicationInfo: Stubbed. {TitleId: 72324500776771584}
00:00:07.982 |S| HLE.OsThread.22 ServiceAcc IsUserAccountSwitchLocked: Stubbed. 
00:00:07.983 |S| HLE.OsThread.22 ServiceNs CheckAddOnContentMountStatus: Stubbed. 
00:00:08.040 |S| HLE.OsThread.9 ServiceHid ActiveNpadImpl: Stubbed. {appletResourceUserId: 1 ; revision: 3}
00:00:08.045 |S| HLE.OsThread.9 ServiceHid SetSupportedNpadIdType: Stubbed. 9 Players: Player1,Player2,Player3,Player4,Player5,Player6,Player7,Player8,Handheld
00:00:08.046 |S| HLE.OsThread.9 ServiceHid SetSupportedNpadStyleSet: Stubbed. {pid: 98 ; appletResourceUserId: 1 ; type: ProController, Handheld, JoyconPair, JoyconLeft, JoyconRight}
00:00:09.652 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair, JoyconLeft, JoyconRight' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:10.899 |S| HLE.OsThread.10 ServiceNv ZbcSetTable: Stubbed. 
00:00:10.900 |S| HLE.OsThread.10 ServiceNv GetActiveSlotMask: Stubbed. 
00:00:11.653 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair, JoyconLeft, JoyconRight' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:13.655 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair, JoyconLeft, JoyconRight' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:13.781 |S| HLE.OsThread.9 ServiceHid ActivateMouse: Stubbed. {appletResourceUserId: 1}
00:00:13.785 |S| HLE.OsThread.9 ServiceHid ActivateKeyboard: Stubbed. {appletResourceUserId: 1}
00:00:15.102 |I| HLE.OsThread.6 AudioRenderer AcquireSessionId: Registered new renderer (0)
00:00:15.105 |I| HLE.OsThread.6 AudioRenderer Initialize: Initializing with REV11
00:00:15.131 |I| AudioProcessor.Worker AudioRenderer Work: Starting audio processor
00:00:15.131 |I| AudioRendererManager.Worker AudioRenderer SendCommands: Starting audio renderer
00:00:15.141 |I| HLE.OsThread.6 AudioRenderer Start: Starting renderer id 0
00:00:15.299 |S| HLE.OsThread.6 ServiceAudio QueryAudioDeviceOutputEvent: Stubbed. 
00:00:15.301 |S| HLE.OsThread.6 ServiceAudio GetActiveChannelCount: Stubbed. 
00:00:15.656 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair, JoyconLeft, JoyconRight' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:16.782 |S| HLE.OsThread.22 ServiceAm SetOperationModeChangedNotification: Stubbed. {operationModeChangedNotification: True}
00:00:16.782 |S| HLE.OsThread.22 ServiceAm SetPerformanceModeChangedNotification: Stubbed. {performanceModeChangedNotification: True}
00:00:16.799 |I| HLE.OsThread.22 ServiceAm SetTerminateResult: Result = 0x00000a14 (ResultKvdbInvalidKeyValue (2020-0005)).
00:00:17.270 |I| HLE.GuestThread.40 Cpu PrintGuestStackTrace: Guest stack trace:
Process: The Legend of Zelda: Tears of the Kingdom, PID: 98
   0x000000000ce58d80	sdk:0x8fd80	
   0x000000000ce58e48	sdk:0x8fe48	
   0x000000000ce58f28	sdk:0x8ff28	
   0x000000000cea4658	sdk:0xdb658	
   0x0000000008e1bb50	main:0xe17b50	
   0x0000000008e1b8f8	main:0xe178f8	
   0x000000000a8364fc	main:0x28324fc	
   0x000000000a33bb98	main:0x2337b98	
   0x000000000aa22608	main:0x2a1e608	
   0x000000000cef3518	sdk:0x12a518	
   0x000000000cef92f4	sdk:0x1302f4	
   0x0000000000000000

00:00:17.277 |I| HLE.GuestThread.40 Cpu PrintGuestRegisterPrintout: Guest CPU registers:
	X[00]:	0x0000000000000000
	X[01]:	0x00000049cfeb1cb0 (SP:-0x01c0)
	X[02]:	0x0000000000000004
	X[03]:	0x0000000000000000
	X[04]:	0x0000000000000000
	X[05]:	0x0000000000000000
	X[06]:	0x0000000000000000
	X[07]:	0x0000000000000000
	X[08]:	0x0000000100000000
	X[09]:	0x000000000cdc9000 (sdk:0x0000)	=> 
	X[10]:	0xfefefefefefefeff
	X[11]:	0x0000008000000000
	X[12]:	0x0000000000000000
	X[13]:	0x0000000000000002
	X[14]:	0x0000000000000000
	X[15]:	0x0000000000000002
	X[16]:	0x000000000ce5a700 (sdk:0x91700)	=> nn::diag::detail::Abort(nn::Result const*)
	X[17]:	0x000000000dacf520
	X[18]:	0x0000000000000000
	X[19]:	0x00000049cfeb1cb0 (SP:-0x01c0)
	X[20]:	0x0000000000000000
	X[21]:	0x000000000d879757
	X[22]:	0x000000000d879757
	X[23]:	0x0000000000000000
	X[24]:	0x000000000d879757
	X[25]:	0x00000049cfeb1c70 (SP:-0x0180)
	X[26]:	0x000000000d879757
	X[27]:	0x000000000ce58e48 (sdk:0x8fe48)	=> 
	X[28]:	0x0000000000000000
	FP:	0x00000049cfeb1af0 (SP)
	LR:	0x000000000ce5a718 (sdk:0x91718)	=> nn::diag::detail::Abort(nn::Result const*):0x0018
	SP:	0x00000049cfeb1af0 (SP)
	PC:	0x000000000cf21494 (sdk:0x158494)	=> 

00:00:17.290 |W| HLE.GuestThread.21 KernelSvc : WaitForAddress() = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.23 KernelSvc : WaitSynchronization(handleIndex: 0xFFFFFFFF) = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.24 KernelSvc : WaitForAddress() = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.36 KernelSvc : WaitForAddress() = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.25 KernelSvc : WaitForAddress() = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.28 KernelSvc : WaitForAddress() = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.30 KernelSvc : WaitForAddress() = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.26 KernelSvc : WaitForAddress() = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.52 KernelSvc : WaitForAddress() = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.31 KernelSvc : WaitForAddress() = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.29 KernelSvc : WaitForAddress() = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.62 KernelSvc : WaitForAddress() = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.32 KernelSvc : WaitForAddress() = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.33 KernelSvc : WaitForAddress() = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.38 KernelSvc : WaitForAddress() = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.34 KernelSvc : WaitForAddress() = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.35 KernelSvc : WaitForAddress() = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.41 KernelSvc : SendSyncRequest() = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.37 KernelSvc : WaitForAddress() = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.54 KernelSvc : WaitForAddress() = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.42 KernelSvc : WaitForAddress() = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.55 KernelSvc : WaitSynchronization(handleIndex: 0xFFFFFFFF) = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.43 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.59 KernelSvc : WaitForAddress() = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.51 KernelSvc : WaitForAddress() = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.60 KernelSvc : WaitForAddress() = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.53 KernelSvc : WaitForAddress() = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.57 KernelSvc : WaitForAddress() = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.58 KernelSvc : WaitForAddress() = TerminationRequested
00:00:17.290 |W| HLE.GuestThread.61 KernelSvc : WaitForAddress() = TerminationRequested
00:00:17.362 |E| HLE.GuestThread.40 Application : Unhandled exception caught: Ryujinx.HLE.Exceptions.GuestBrokeExecutionException: The guest program broke execution!
   at Ryujinx.HLE.HOS.Kernel.SupervisorCall.Syscall.Break(UInt64 reason) in D:\a\Ryujinx\Ryujinx\src\Ryujinx.HLE\HOS\Kernel\SupervisorCall\Syscall.cs:line 1924
   at Ryujinx.HLE.HOS.Kernel.SupervisorCall.SyscallDispatch.Break64(Syscall syscall, IExecutionContext context) in D:\a\Ryujinx\Ryujinx\src\Ryujinx.HLE\Ryujinx.Horizon.Kernel.Generators\Ryujinx.Horizon.Kernel.Generators.SyscallGenerator\SyscallDispatch.g.cs:line 1807
   at Ryujinx.HLE.HOS.Kernel.SupervisorCall.SyscallDispatch.Dispatch64(Syscall syscall, IExecutionContext context, Int32 id) in D:\a\Ryujinx\Ryujinx\src\Ryujinx.HLE\Ryujinx.Horizon.Kernel.Generators\Ryujinx.Horizon.Kernel.Generators.SyscallGenerator\SyscallDispatch.g.cs:line 3569
   at Ryujinx.HLE.HOS.Kernel.SupervisorCall.SyscallHandler.SvcCall(IExecutionContext context, UInt64 address, Int32 id) in D:\a\Ryujinx\Ryujinx\src\Ryujinx.HLE\HOS\Kernel\SupervisorCall\SyscallHandler.cs:line 17
   at ARMeilleure.Instructions.NativeInterface.SupervisorCall(UInt64 address, Int32 imm) in D:\a\Ryujinx\Ryujinx\src\ARMeilleure\Instructions\NativeInterface.cs:line 50
   at ARMeilleure.Translation.Translator.Execute(ExecutionContext context, UInt64 address) in D:\a\Ryujinx\Ryujinx\src\ARMeilleure\Translation\Translator.cs:line 155
   at Ryujinx.Cpu.Jit.JitCpuContext.Execute(IExecutionContext context, UInt64 address) in D:\a\Ryujinx\Ryujinx\src\Ryujinx.Cpu\Jit\JitCpuContext.cs:line 32
   at Ryujinx.HLE.HOS.ArmProcessContext`1.Execute(IExecutionContext context, UInt64 codeAddress) in D:\a\Ryujinx\Ryujinx\src\Ryujinx.HLE\HOS\ArmProcessContext.cs:line 60
   at Ryujinx.HLE.HOS.Kernel.Threading.KThread.ThreadStart() in D:\a\Ryujinx\Ryujinx\src\Ryujinx.HLE\HOS\Kernel\Threading\KThread.cs:line 1264

### OS

Windows 11

### Ryujinx version

1.1.962

### Game version

1.2.0

### CPU

i5-10400f

### GPU

GTX 1650 Super

### RAM

16GB

### List of applied mods

None.

**Comment by AcK77** [comment]
`ResultKvdbInvalidKeyValue` means your game (or update) dump is wrong, redump it and try again.
Additionally, your controller settings are wrong too.
I suggest you to come to our Discord for support, Thanks.

**Comment by snl25** [comment]
> `ResultKvdbInvalidKeyValue` means your game (or update) dump is wrong, redump it and try again. Additionally, your controller settings are wrong too. I suggest you to come to our Discord for support, Thanks.

Thanks for your help. I finally got it to boot. It wasn't the dump but something else in the Ryujinx folder. When Googling the 'ResultKvdbInvalidKeyValue' error it brought me to the Official Ryujinx FAQ. It may have been something to do with the Save Data being corrupt. I deleted the save data folder for TOTK, but that didn't initially work. So I deleted the entire Ryujinx data folder as per the instructions. That did the trick and allowed TOTK to boot.

Thanks again!
